### PR TITLE
0167-app-services (Accepted With Revisions)

### PR DESCRIPTION
**Pull request by JackLivio**
Make revisions as highlighted [here](https://github.com/smartdevicelink/sdl_evolution/issues/675)

```
The Steering Committee voted to accept this proposal with revisions. The author is to revert the change of renaming `AppServiceCapability` params in the `SystemCapability` struct to align with how the proposal was originally accepted.

* It was also noted that the only AppServiceTypes that will be included in the Q1 releases are `MEDIA`, `WEATHER` and `NAVIGATION`.
* ` The VOICE_ASSISTANT service data should be removed from the main proposal and put into a new section titled “Amendments” in the proposal with a note that it was removed from the first proposal.
* GENERIC, COMMUNICATION_VOIP, MESSAGING and TTS should be removed from this proposal as they will not be implemented in this iteration. When these specific AppServiceTypes are to be added, they will need to be submitted in separate proposals and voted on.
```

### CODE REVIEW

File: proposals/0167-app-services.md [hunk 1]
```diff
@@ -978,44 +903,41 @@ The HMI will be responsible for the actual selection and activation of app servi
...
 
 <function name="AppServiceActivation" messagetype="request">
-  <param name="serviceId" type="String" mandatory="true" >
+  <param name="serviceID" type="String" mandatory="true" >
     <description>The ID of the service that should have an activation event take place on</description>
   </param>
-  <param name="serviceType" type="Common.ServiceType" mandatory="true">
-    <description>The service type the service should have the activation event occur on</description>
+  <param name="serviceType" type="String" mandatory="true">
+    <description>The type of service that is to be offered by this app. See AppServiceType for known enum equivalent types. Parameter is a string to allow for new service types to be used by apps on older versions of SDL Core.</description>
```
Comment (jacobkeeler) [inline proposals/0167-app-services.md]: Should be removed, as was done in the response.

File: proposals/0167-app-services.md [hunk 2]
```diff
@@ -978,44 +903,41 @@ The HMI will be responsible for the actual selection and activation of app servi
 
 ```xml
 <function name="GetAppServiceRecords" messagetype="request">
-  <param name="serviceTypes" type="Common.ServiceType" mandatory="false">
-    <description>If included, only service records of supplied type will be returned in response. If not included, all service records for all types will be returned.</description>
+  <param name="serviceType" type="String" mandatory="false">
+    <description>The type of service that is to be offered by this app. See AppServiceType for known enum equivalent types. Parameter is a string to allow for new service types to be used by apps on older versions of SDL Core.</description>
```
Comment (jacobkeeler) [inline proposals/0167-app-services.md]: Description doesn't match for this RPC

Comment (JackLivio) [reply proposals/0167-app-services.md]: Fixed

File: proposals/0167-app-services.md (hunk 1 above)
Comment (JackLivio) [reply proposals/0167-app-services.md]: Updated

**Pull request merged**